tests/test_documentation_build.py

```diff
@@ -6,9 +6,3 @@ class TestDocumentation:
     def test_build(self, tmp_path: Path) -> None:
         """Test the build of the main `govcookiecutter` repository."""
         assert main(["-b", "html", "docs", str(tmp_path.joinpath("_build"))]) == 0
-
-    def test_external_links(self, tmp_path: Path) -> None:
```

Comment:
As above, is there a documented explanation for removal or plan to replace this?

Reply:
Sounds good 🙂 Do we have a changelog or are changes communicated to users another way?

Reply:
I have communicated to users on slack what the changes would be but there is no chnagelog. I would be happy to include a changelog next but as this fixes an issue which stops users from using govcookiecutter in some incidences I would like to push it through first. Happy to be disagreed with though

Reply:
Think that's fine for this fix. I think it's worth asking users how they'll like to be informed about future changes